config/src/test/kotlin/maru/config/HopliteFriendlinessTest.kt

```diff
@@ -67,6 +67,7 @@ class HopliteFriendlinessTest {
               staticPeers = emptyList(),
               reconnectDelay = 500.milliseconds,
             ),
+          p2pConfig = P2P(ipAddress = "127.0.0.1", port = "3322", discoveryPort = "9876", staticPeers = emptyList()),
```

Comment:
Can you please add bootnodes setting somewhere here? Because right now it's not tested at all

Reply:
Added a bootnode